Snowdog_Components/components/Molecules/form/datepicker/_datepicker.scss

```diff
@@ -0,0 +1,89 @@
+@import 'datepicker-variables';
+
+.ui-datepicker {
+    border: $datepicker__border;
+    background-color: $datepicker__background-color;
+
+    .ui-datepicker-prev {
+        top: $datepicker__arrow-top;
+
+        .ui-icon {
+            border-color: $datepicker__prev-icon-border-color;
+        }
+    }
+
+    .ui-datepicker-next {
+        top: $datepicker__arrow-top;
+
+        .ui-icon {
+            border-color: $datepicker__next-icon-border-color;
+        }
+    }
+
+    .ui-datepicker-title {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: row-reverse;
```

Comment:
It would be good to add here variables (at least for flex-direction and justify-content and align-items)

Reply:
We also need to add changelog info with info "Affects: [area/pages/elements]" - so it will be easier for some people to see what they should test after upgrade